avocado/formatters.py

```diff
@@ -287,7 +295,7 @@ def get_meta(self, exporter=None):
 
         return meta
 
-    def to_string(self, value, **context):
+    def to_string(self, value, field, context):
```

Comment:
Ah, ok. I mistook it to be a `DataContext` which seemed unused. This is fine as is.